lamindb_setup/_check_setup.py

```diff
@@ -70,14 +71,48 @@ def _get_current_instance_settings() -> InstanceSettings | None:
         return None
 
 
+def _check_in_modules(module: str, isettings: InstanceSettings | None = None) -> None:
+    not_in_instance_msg = f"'{module}' is missing from this instance."
+
+    if isettings is not None:
+        if module not in isettings.modules:
+            raise RuntimeError(not_in_instance_msg)
+        else:
+            return
+
+    from django.apps import apps
+
+    for app in apps.get_app_configs():
+        if module == app.name:
+            return
+    raise RuntimeError(not_in_instance_msg)
+
+
+def _module_name() -> str | None:
```

Comment:
A more descriptive name and/or a comment, please.

Reply:
done